### ISSUE TITLE
IndexError: list index out of range : max_queue_length = tmp_element[1]

### ISSUE THREAD

**Issue by dkasprick** [ilbuonmarcio/TurnipMaster#2]
Seems to happen when I use the max_queue_length. Tried to use --max-queue-length 10 or --max-queue-length 15

```
Traceback (most recent call last):
  File "turnipexchange.py", line 245, in <module>
    islands = tex.get_islands()
  File "turnipexchange.py", line 50, in get_islands
    max_queue_length = tmp_element[1]
IndexError: list index out of range
```

**Comment by ilbuonmarcio** [comment]
Hi! This is probably linked to issue #3, in which Patreon default selection defaults to no islands... I'll have a look tomorrow and let you know once the fix is available!

**Comment by dkasprick** [comment]
Hi Yep thanks I think they are related. I noticed that when Patreon is
defaulted and there are no islands it creates a dummy island which probably
causes the error. I did a pull request, never used selenium or anything
before though so feel free to correct anything. I got it to work locally
for myself

**Comment by ilbuonmarcio** [comment]
I just checked your code and it looks fine, so I'll wait for the pull request from you!

**Comment by ilbuonmarcio** [comment]
Fixed as per #4
